clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -440,4 +448,8 @@ def InlineAttr : CIR_Attr<"Inline", "inline"> {
   }];
 }
 
+def OptnoneAttr : UnitAttr<"Optnone", "optnone"> {
```

Comment:
Can you rename `OptnoneAttr` to `OptNoneAttr`? That will make it more consistent with other uses.